src/utils.py

```diff
@@ -81,6 +81,13 @@ def get_file_mtime(string):
 def is_file_dir(string):
     return os.path.isdir(string)
 
+@cached
+def get_file_icon_name(string):
+    file = Gio.file_new_for_path(string)
+    fileinfo = file.query_info(Gio.FILE_ATTRIBUTE_STANDARD_ICON, 0)
+    fileicon = fileinfo.get_icon().get_names()[1]
```

Comment:
This is wrong, [1] is arbitrary.

Why even use names here instead of just the GIcon directly?